charts/argo-cd/templates/argocd-server/ingress.yaml

```diff
@@ -29,36 +29,27 @@ spec:
   rules:
   {{- if .Values.server.ingress.hosts }}
     {{- range $host := .Values.server.ingress.hosts }}
+    {{- if and $.Values.server.ingressGrpc.isAWSALB $.Values.server.ingressGrpc.enabled }}
+      {{ $serviceName = printf "%s-grpc" (include "argo-cd.server.fullname" $) }}
```

Comment:
This will break AWS integration. AWS ALB requires 2 ingress routes (1st for http / https, 2nd for grpc traffic targeting grpc service with ALB annotations)

Reply:
I was not aware of this. @pdrastil thanks for review, I will cancel this PR